### PR TITLE
Quick fix for the create workspace page

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Quick fix for the create workspace page:
 - prevent page refresh when when we click Enter;
 - add a placeholder;
 - rename routes;

Screenshot from 2021-05-24 14-58-12

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/18975

### CODE REVIEW

File: src/pages/GetStarted/GetStartedTab/index.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class SamplesListTab extends React.PureComponent<Props, State> {
           style={{ background: '#f0f0f0' }}
         >
           <PageSection variant={PageSectionVariants.light}>
-            <Form>
+            <Form onSubmit={e => e.preventDefault()}>
```
Comment (sleshchenko) [inline src/pages/GetStarted/GetStartedTab/index.tsx:86]: As far as I understand with this click on Enter will do nothing.
Ideally, it should emulate the click on Create&Open button.

**Review by sleshchenko — APPROVED** [review]

**Review by akurinnoy — APPROVED** [review]

File: src/pages/GetStarted/GetStartedTab/index.tsx (hunk 1 above)
Comment (olexii4) [reply src/pages/GetStarted/GetStartedTab/index.tsx]: I have fixed it.

**Review by sleshchenko — APPROVED** [review]
Tested and now Enter click works just fine!

**Comment by olexii4** [comment]
[dashboard-ci-test]

**Pull request merged**